app/src/main/java/org/mozilla/focus/state/AppAction.kt

```diff
@@ -105,4 +105,9 @@ sealed class AppAction : Action {
      * State of show Tracking Protection CFR has changed
      */
     data class ShowTrackingProtectionCfrChange(val value: Map<String, Boolean>) : AppAction()
+
+    /**
+     * State of Snackbar for promote search widget has changed
+     */
+    data class ShowSearchWidgetSnackBar(val value: Boolean) : AppAction()
```

Comment:
Seems to me like `AppAState` tends to grow with flags only used once. Don't think this is a good approach going forward, maybe we could try splitting the state in smaller others? Food for thought.